src/server/src/main/java/io/cassandrareaper/storage/postgresql/LongCollectionSqlType.java

```diff
@@ -0,0 +1,39 @@
+/*
+ *
+ * Copyright 2017-2018 The Last Pickle Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.cassandrareaper.storage.postgresql;
+
+import java.util.Collection;
+
+import com.google.common.collect.Lists;
+
+/**
+ * This is required to be able to map in generic manner into Postgres array
+ * types through JDBI.
+ */
+public final class LongCollectionSqlType {
```

Comment:
this class can't be in this package, as it then creates a package tangle. 
Other classes in Reaper do not reference this package, with the exception of `PostgresStorage`, which it is private to.

could you please move it back to being an inner class in `RepairSchedule`.